### PR TITLE
Add the possibility to bind multiple routing keys

### PR DESCRIPTION
**Pull request by johnduro**
Hi,

I adapted the work of @sterrien in ricbra/rabbitmq-cli-consumer#59 adding the possibility to bind multiple routing key to a queue. I added test to it.

Tell me if you want any modification, i'll be glad to do them !

Thanks !

### CODE REVIEW

File: consumer/connection_test.go [hunk 1]
```diff
@@ -63,6 +63,32 @@ const ttlConfig = `[rabbitmq]
   error=a
   info=b`
 
+const multipleRoutingKeysConfig = `[rabbitmq]
```
Comment (corvus-ch) [inline consumer/connection_test.go]: Recent changes make most of the config values optional. Please remove all config keys which are not required for this test.

File: config/config.go [hunk 2]
```diff
@@ -111,9 +111,9 @@ func (c Config) MessageTTL() int32 {
 	return int32(c.QueueSettings.MessageTTL)
 }
 
-// RoutingKey returns the configured key for message routing.
-func (c Config) RoutingKey() string {
-	return transformToStringValue(c.QueueSettings.Routingkey)
```
Comment (corvus-ch) [inline config/config.go]: This is a potential BC brake. Please add another test which contains a single `routingkey` set to `"<empty>"` and verify that the queue is bound with an empty string (`""`).

Comment (johnduro) [reply config/config.go]: you were right, i corrected this and added a test.

I also added another test for when routing key is not defined. Don't hesitate to tell me if you don't like the change.

File: consumer/connection_test.go (hunk 1 above)
Comment (johnduro) [reply consumer/connection_test.go]: I deleted most of the config, tell me if it is what you had in mind.

**Comment by corvus-ch** [comment]
Thanks @johnduro, your changes are fine. Codeclimate has spotted an issue to be fixed with `go fmt`. The other complaints I am willing to ignore. And because #13 is now merged, this PR is now in conflict. Can you please resolve this two issues.

**Comment by johnduro** [comment]
hey @corvus-ch,

I updated my pr but i have errors in Travis CI i cannot reproduce in local, do you have them too ?

**Comment by corvus-ch** [comment]
Yes I have them but only occasionally. They occur more often on TravisCi. The error is, that empty lines appear in the output during integration tests. So far I was not able to trace them down and I was able to circumvent the issue by rerun the build jobs.

**Comment by corvus-ch** [comment]
Thanks @johnduro.

**Comment by johnduro** [comment]
Ah we got those random errors sometimes on circle-ci too, anyway thanks for the 2 PRs !
